src/pages/schedule.js

```diff
@@ -4,90 +4,29 @@ import Helmet from 'react-helmet';
 import MarkdownStyles from '../css/markdown-styles';
 import { Container, SmallTitle, MarkdownContainer } from '../utils/components';
 import Layout from '../utils/components/Layout';
-
-const currentDate = new Date();
+import { groupBy } from '../utils/groupBy';
 
 export default function Schedule({ data }) {
-  const { children } = data.transcripts.nodes.reduce(
-    (acc, node) => {
-      const {
-        childMarkdownRemark: {
-          html,
-          frontmatter: {
-            date,
-            dateGroup,
-            dateString,
-            description,
-            location,
-            people,
-            recording,
-            time,
-            title,
-          },
-        },
-        name,
-        relativeDirectory,
-      } = node;
-
-      const isUpcoming = currentDate < new Date(date);
+  const currentDate = new Date();
```

Comment:
Moved this on the off chance somebody leaves the site open for multiple days across when a Q&A happens. I think that would have rendered incorrectly until refresh because the date would only be evaluated at initial load, vs on page render with this change.

I suppose that this is gonna cause a flash/mismatch as date drifts away from the last server build, but since we'll have to build to add new events that feels acceptable.